tests/unit/gapic/trace_v2/test_trace_service.py

```diff
@@ -18,6 +18,7 @@
 
 import grpc
 from grpc.experimental import aio
+import json
```

Comment:
Adding this import seems pointless (and lint-breaking) without also adding code which uses them.